### ISSUE TITLE
Get profit percentage for Binary and Digital Spot Options 

### ISSUE THREAD

**Issue by shrisheshpratik** [Lu-Yi-Hsun/iqoptionapi#318]
Hi!
I want to compare payout percentage of a particular asset in binary and digital spot (assuming it is open in both binary and digital). Anyone can help me how can I do so?

Eg- 
 
In digital options, payout of EURUSD for 1 minute is 87%
![image](https://user-images.githubusercontent.com/68203349/140288185-3e379d1d-95cd-4cdb-ab6f-f33d0def88aa.png)

But in binary options it is 80%
![image](https://user-images.githubusercontent.com/68203349/140289166-23a17811-a5e6-4c9d-ab11-ff44380d63c2.png)

So basically I want to get both these profit rates so that i can compare them

**Comment by wujileee** [comment]
`def payout(par, tipo, timeframe = 1):
	if tipo == 'turbo':
		todos_os_pares = API.get_all_profit()
		return int(100 * todos_os_pares[par]['turbo'])
		
	elif tipo == 'digital':
	
		API.subscribe_strike_list(par, timeframe)
		while True:
			payout_digital = API.get_digital_current_profit(par, timeframe)
			if payout_digital != False:
				payout_digital = int(payout_digital)
				break
			time.sleep(1)
		API.unsubscribe_strike_list(par, timeframe)
		return payout_digital


def qual_tipo_paga_mais(paridade):
        payout_turbo = payout(paridade, 'turbo')     
        payout_digital = payout(paridade, 'digital')
        print('payout turbo ', payout_turbo)
        print('payout digital ', payout_digital)
        if payout_turbo >= payout_digital:
            tipo = 'turbo'
        else:
            tipo = 'digital'
        return tipo`
